lib/files.test.js

```diff
@@ -335,4 +335,31 @@ describe(_.startCase(filename), function () {
       expect(fn(name)).to.equal(result);
     });
   });
-});
+
+  describe('readFilePromise', function () {
+    const fn = lib[this.title];
+
+    it('returns file contents', function () {
+      const result = 'result',
+        name = 'article.css';
+
+      sandbox.stub(fs, 'readFile', function (path, x, callback) {
```

Comment:
Yeah, this is weird, sorry about that. The actual [`fs.readFile`](https://nodejs.org/api/fs.html#fs_fs_readfile_file_options_callback) function accepts an optional options object **or** string as a second argument (say that five times fast). In the actual function I pass `utf-8` as the encoding option, so for the test Morgan and I figured out we just needed a placeholder. I could name that differently though. I think we were just trying to debug the tests quickly and I forgot to re-name.